jupyter-pytorch-full/rockcraft.yaml

```diff
@@ -141,20 +148,47 @@ parts:
       "${CONDA_BIN}/conda" clean -a -f -y
 
       # Install the jupyter python requirements
+      echo "jupyterlab ==${JUPYTERLAB_VERSION}" >> ${CONDA_DIR}/conda-meta/pinned
+      echo "notebook ==${JUPYTER_VERSION}" >> ${CONDA_DIR}/conda-meta/pinned
+      "${CONDA_BIN}/conda" install -y -q \
+        jupyterlab==${JUPYTERLAB_VERSION} \
+        notebook==${JUPYTER_VERSION}
       cp ${CRAFT_PART_SRC}/components/example-notebook-servers/jupyter/requirements.txt jupyter-requirements.txt
       "${CONDA_BIN}/pip" install --no-cache-dir -r jupyter-requirements.txt
 
       # Generate a jupyter lab config
       "${CONDA_BIN}/jupyter" lab --generate-config
+      "${CONDA_BIN}/jupyter" notebook --generate-config
+      "${CONDA_BIN}/jupyter" labextension disable "@jupyterlab/apputils-extension:announcements"
 
       # Install the jupyter-pytorch cpu requirements
-      cp $CRAFT_PART_SRC/components/example-notebook-servers/jupyter-pytorch/cpu-requirements.txt cpu-requirements.txt
-      "${CONDA_BIN}/pip" install --no-cache-dir -r cpu-requirements.txt
+      "${CONDA_BIN}/pip" install --quiet --no-cache-dir --index-url https://download.pytorch.org/whl/cpu --extra-index-url https://pypi.org/simple \
+        torch==${PYTORCH_VERSION} \
+        torchvision==${TORCHVISION_VERSION} \
+        torchaudio==${TORCHAUDIO_VERSION}
+      cp $CRAFT_PART_SRC/components/example-notebook-servers/jupyter-pytorch/requirements.txt requirements.txt
+      "${CONDA_BIN}/pip" install --no-cache-dir -r requirements.txt
 
       # Install the jupyter-pytorch-full cpu requirements
```

Comment:
Should we omit cpu from wording in order to follow upstream naming?

Reply:
fixed